File: templates/java.junit/%Kata_file%Test.java
// Adapt the code to your code kata %kata_file%.

import org.junit.*;
import static org.junit.Assert.*;

public class %Kata_file%Test {
	
	@Test
	public void testFoo() {
		%Kata_file% object_under_test = new %Kata_file%();
		assertEquals("foo", object_under_test.foo());
	}
	
}

class %Kata_file% {

	public String foo() {
		return "fixme";
	}
	
	
}

File: templates/scala.junit/%Kata_file%Test.scala
// Adapt the code to your code kata primes.

import org.junit.Test
import org.junit.Assert._

class %Kata_file%Test {

  @Test
  def verifyThatFooIsFixed() = {
    val objectUnderTest = new %Kata_file%()
    assertEquals("foo", objectUnderTest.foo())
  }

}

private class %Kata_file% {

  def foo() = {
    "fixme"
  }

}
